newsletter/src/utils/cors.ts

```diff
@@ -0,0 +1,16 @@
+import { ORIGIN } from './constants';
+
+const MATCHES: RegExp[] = [
+  /deploy-preview-[0-9]+--koda-canary.netlify.app/,
+  /deploy-preview-[0-9]+--nuxt-kodadot.netlify.app/,
+  /kodadot.xyz/,
```

Comment:
missing `beta.`,`nuxt.` & `canary.`?

Reply:
`/kodadot.xyz/` will match all. It is the file that we are copypasting over and over ;D

Reply:
yeah regex my bad ..